packages/babel-helpers/src/helpers.js

```diff
@@ -1053,3 +1053,12 @@ helpers.classPrivateFieldSet = helper("7.0.0-beta.0")`
     return value;
   }
 `;
+
+helpers.declareNilProps = helper("7.0.0-beta.0")`
```

Comment:
This should be `7.0.1`, since it represents the first version where the helper is available.